internal/util/yamlutil/scanner.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Modified from github.com/kubernetes-sigs/controller-tools/pkg/util/util.go
```

Comment:
What is modified and taken from there? Not sure how this works by apache license, but maybe we need to a more detailed comment about what was copied from there?

Reply:
That comment doesn’t pertain to these additions, will remove.